webpack.config.js

```diff
@@ -0,0 +1,63 @@
+// const path = require('path');
+// const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+// module.exports = {
+//   mode: 'development',
+//   entry: './src/index.js',
+//   output: {
+//     filename: 'bundle.js',
+//     path: path.resolve(__dirname, 'dist'),
+//     clean: true,
+//   },
+//   module: {
+//     rules: [
+//       {
+//         test: /\.css$/i,
+//         use: ['style-loader', 'css-loader'],
+//       },
+//     ],
+//   },
+//   plugins: [
+//     new HtmlWebpackPlugin({
+//       title: 'ToDo-List',
+//       filename: 'index.html',
+//     }),
+//   ],
+// };
```

Comment:
- Kindly, Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍